.github/workflows/nix.yml

```diff
@@ -0,0 +1,39 @@
+name: nix
+on: [pull_request, push]
+
+jobs:
+  update-deps:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v1
+
+    - name: Set up nix
+      uses: cachix/install-nix-action@v3
+
+    - name: Update deps.nix
+      run: nix-shell -p vgo2nix --command vgo2nix
+
+    - name: Create Pull Request
+      if: github.event_name == 'push'
+      uses: peter-evans/create-pull-request@v1.6.0
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        COMMIT_MESSAGE: "Update deps.nix"
+        PULL_REQUEST_TITLE: "Update deps.nix"
+        PULL_REQUEST_BODY: "Auto-generated pull request: Keep `deps.nix` up to date with `go.mod`"
+
+    - name: Fail on outdated deps.nix
+      run: git diff --exit-code
+
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v1
+
+    - name: Set up nix
+      uses: cachix/install-nix-action@v3
+
+    - name: Build
+      run: nix-build
```

Comment:
Awesome use of GitHub Actions ❤️

Reply:
Well, nix is awesome. In combination with Actions its just pure magic :rocket:
Love both things :heart: